common/templates/common/sample_djangre.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'base_templates/base.html' %}
+{% load django_react_components %}
+{% load render_bundle from webpack_loader %}
+{% block head %}
+    {% render_bundle 'runtime' %}
+    {% render_bundle 'Home' %}
+    {% render_bundle 'Hello' %}
+{% endblock %}
+
+{% block body %}
+  {% react_component 'Home' id='home' message=message %}
+  {% react_component 'Hello' id='hello' msg=hello_msg %}
```

Comment:
If I'm remembering correctly how to use this, I think it might also be good to give an example of using `props` instead of individual kwargs.